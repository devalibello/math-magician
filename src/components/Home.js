import React from 'react';
import style from '../styles/Home.module.css';

const Home = () => (
  <div className={style.pageContainer} data-testid="page-container">
    <h1 className={style.homePage}>Welcome to our home page!</h1>
    <p className={style.text}>
      Lorem ipsum dolor sit, amet consectetur adipisicing elit.
      Accusantium omnis repellat, doloribus, nemo quas ipsam sequi
      iusto ex dicta dolorum eos nulla eligendi ut maxime ipsum distinctio
      quia fugiat cum!. Lorem ipsum dolor sit amet consectetur adipisicing elit.
      Excepturi, officia harum voluptate reprehenderit itaque earum. Ex sapiente ea eum
      eos quod hic doloribus corporis officiis magni. Tempore assumenda dolores quaerat!
    </p>
    <br />
    <p className={style.text}>
      Lorem ipsum dolor sit, amet consectetur adipisicing elit.
      Accusantium omnis repellat, doloribus, nemo quas ipsam sequi
      iusto ex dicta dolorum eos nulla eligendi ut maxime ipsum distinctio
      quia fugiat cum!. Lorem ipsum dolor sit amet consectetur adipisicing elit.
      Fuga sed architecto doloribus minus aperiam non dicta, quod modi officiis
      ipsum, impedit ab beatae rem eius sint enim est. Adipisci, autem?
    </p>
  </div>
);

export default Home;
