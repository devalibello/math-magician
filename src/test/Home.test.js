import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import Home from '../components/Home';

it('Home render correctly', () => {
  const tree = renderer.create(
    <MemoryRouter>
      <Home />
    </MemoryRouter>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
