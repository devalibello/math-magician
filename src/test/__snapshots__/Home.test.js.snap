// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home render correctly 1`] = `
<div
  className="pageContainer"
  data-testid="page-container"
>
  <h1
    className="homePage"
  >
    Welcome to our home page!
  </h1>
  <p
    className="text"
  >
    Lorem ipsum dolor sit, amet consectetur adipisicing elit. Accusantium omnis repellat, doloribus, nemo quas ipsam sequi iusto ex dicta dolorum eos nulla eligendi ut maxime ipsum distinctio quia fugiat cum!. Lorem ipsum dolor sit amet consectetur adipisicing elit. Excepturi, officia harum voluptate reprehenderit itaque earum. Ex sapiente ea eum eos quod hic doloribus corporis officiis magni. Tempore assumenda dolores quaerat!
  </p>
  <br />
  <p
    className="text"
  >
    Lorem ipsum dolor sit, amet consectetur adipisicing elit. Accusantium omnis repellat, doloribus, nemo quas ipsam sequi iusto ex dicta dolorum eos nulla eligendi ut maxime ipsum distinctio quia fugiat cum!. Lorem ipsum dolor sit amet consectetur adipisicing elit. Fuga sed architecto doloribus minus aperiam non dicta, quod modi officiis ipsum, impedit ab beatae rem eius sint enim est. Adipisci, autem?
  </p>
</div>
`;
